ompi/mca/mtl/ofi/mtl_ofi_component.c

```diff
@@ -543,7 +537,7 @@ static int ompi_mtl_ofi_init_regular_ep(struct fi_info * prov, int universe_size
                 return ret;
             }
     }
-#endif /* OPAL_CUDA_SUPPORT && FI_OPT_FI_HMEM_P2P */
+#endif /* HAVE_DECL_FI_OPT_FI_HMEM_P2P */
```

Comment:
For lack of a better place to leave this note, you need to do something with the OFI MTL's configure.m4 as well.

Reply:
This note appears to still apply.

Reply:
Done